<launch>
  <node name="param_test_node" pkg="cpt_ros" type="param_test_node" output="screen">
  <param name="zero_angle" value="0.4"/>
   <param name="zero_x" value="0.0"/>
   <param name="zero_y" value="0.0"/>
   <param name="zero_z" value="0.707"/>
   <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/half_sphere2.off" />
   </node>



</launch>
